Client/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift

```diff
@@ -376,7 +410,7 @@ class TabLocationView: UIView {
   }
   
   fileprivate func updateTextWithURL() {
-    (urlTextField as? DisplayTextField)?.hostString = url?.host ?? ""
+    (urlTextField as? DisplayTextField)?.hostString = url?.withoutWWW.host ?? ""
```

Comment:
This line was actually a bit bugged prior to this change and became obvious during my tests with really large fonts. The host provided was something like `www.amazon.com` but the visible text was `amazon.com`, so the x origin was being altered slightly too much when the two were not matching